### PR TITLE
Update collab-cycle-feedback-staging.md

### LINKED ISSUE #40506: Edit and reorganize the  "VFS Team Tasks to Complete" in Staging Review feedback template
## User Story
As a VFS team member, I want to know what actions to take post Staging Review so that I can ensure I am doing everything that is expected of me.

Assignee: @allison0034 
Peer Reviewer: @CherylEvans 

## Description
At the 10-10EZ Short Form staging Review, there were questions from the VFS teams about the feedback reported by Governance team. Specifically, the VFS team member did not believe the issue documented was the responsibility of her team to fix. 

The purpose of this ticket is to provide additional guidance in the Staging Review feedback template to the VFS team so they know what action to take if they believe the ticket is incorrectly assigned.

The secondary purpose of this ticket is to restructure the _VFS Team Tasks to Complete_ from a checklist to a bulleted list. The heading should also be updated to better reflect what VFS teams need to know, as it is not necessarily a task list for them to take action on.

## Impacted Artifacts 
DraftingTemplate google sheet
Collab Cycle Feedback for Staging Review gh template

## Tasks
**Collab Cycle Feedback for Staging Review gh template:**
- [x] Rename heading from _VFS Team Tasks to Complete_ to _VFS Guidance_
   - Or whatever we think is appropriate. I would just like to emphasize that these aren't required tasks, but steps they should take IF there's uncertainty
- [x] Change check boxes to bullets
- [x] Split, "Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket." into 2 bullets.
- [x] Add a 4th bullet - "If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up."
    - Again, or whatever we think is appropriate. 
- [x] Add a 5th bullet - ~~"Governance team has been advised by our OCTO leadership to flag any issue we find, whether it is in scope for your iteration or not. It is possible that this issue may be out of scope for your iteration, however it is still your responsibility to resolve the issue."~~ Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
   - wordsmith
- [x] All bullets should be written as "you" and not "the VFS team"
- [x] Reorder the bullets as appropriate

**DraftingTemplate google sheet:**
- [ ] The same changes are made to the DraftingTemplate.
Note: Please make changes to the Collab Cycle Feedback for Staging Review gh template first, have them peer reviewed, and then Shira will make the necessary changes to the DraftingTemplate.

## Acceptance Criteria
- [x] VFS teams know how to escalate concerns with the identified Staging Review issue
- [x] VFS teams are no longer asked to check off tasks they have or have not completed

## How to prepare this issue
- [x] Attach story to appropriate Initiative or Quarterly Tech-Debt Epic.
- [x] Add User Story, description, tasks, and Acceptance Criteria
- [x] Add any [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization).
- [x] Once all above items are checked, add the 'Ready' label.
- [x] Ticket has been discussed as a team at Planning.
- [x] Ticket has been sized as a team.
- [x] Ticket has been assigned to a Sprint.
- [x] Once assigned to the upcoming or current Sprint, move to the Planned pipeline.
- [x] Once assigned to the upcoming or current Sprint, add assignee and specify Peer Reviewer.

### PR DESCRIPTION
**Pull request by allison0034**

### CODE REVIEW

**Review by CherylEvans — APPROVED** [review]
@allison0034 All the tasks on the ticket have been addressed in this PR.  Approved. :-)

File: .github/ISSUE_TEMPLATE/collab-cycle-feedback-staging.md [hunk 1]
```diff
@@ -54,10 +54,13 @@ WCAG, Design System, Content Style Guide, a previous gh ticket, etc.
...
 --> 
 
-### VFS Team Tasks to Complete
+### VFS Guidance
 
-- [ ] Comment on the ticket if there are questions or concerns
-- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket. 
+- Comment on the ticket if there are questions or concerns
+- Close the ticket when the issue has been resolved or validated by your Product Owner
+- If your team has additional questions or needs Platform help validating the issue, please comment in the ticket
```
Comment (shiragoodman) [inline .github/ISSUE_TEMPLATE/collab-cycle-feedback-staging.md:61]: can you change this to "comment on the ticket" , not in

File: .github/ISSUE_TEMPLATE/collab-cycle-feedback-staging.md [hunk 2]
```diff
@@ -54,10 +54,13 @@ WCAG, Design System, Content Style Guide, a previous gh ticket, etc.
...
+- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.
+- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
```
Comment (shiragoodman) [inline .github/ISSUE_TEMPLATE/collab-cycle-feedback-staging.md:63]: I think we should switch lines 62 and 63. I think it makes more sense to warn them of potentially out of scope issues and that it's their responsibility, and THEN say that if they don't believe it's their responsibility to fix, etc.

**Review by shiragoodman — CHANGES_REQUESTED** [review]

**Review by allison0034 — COMMENTED** [review]
Yes!

**Comment by allison0034** [comment]
@shiragoodman do you think we need both of these bullets?

- Comment on the ticket if there are questions or concerns

- If a team has additional questions or needs Platform help validating the issue, please comment on the ticket.

**Comment by shiragoodman** [comment]
@allison0034 yea, you're right. They seem to overlap. I would just stick with the 2nd bullet since it's more descriptive. And then move it to the first bullet slot.

**Pull request merged**